frontend/src/api/usePutData.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook updating data.
+ * @param {string} url - adress of request
+ * @param {state} requestBody - body od request made by user
+ * @returns { string object bool }
+ */
+const usePutData = async (url, requestBody) => {
+  const [data, setData] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    async function putData() {
+      try {
+        setIsLoading(true);
+        const response = await fetch(`${url}`, {
+          method: 'PUT',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify(requestBody),
+        });
+        setData(response);
```

Comment:
As we expect the server to return updated resources in the body of the request we should 
check if the response is ok and then setData with the body of the response (but first we need to response.json() because we are using fetch API)